src/TLuaInterpreter.cpp

```diff
@@ -1109,43 +1109,43 @@ int TLuaInterpreter::addMapEvent(lua_State * L){
     QStringList actionInfo;
     if( ! lua_isstring( L, 1 ) )
     {
-        lua_pushstring( L, "addMapEvent: wrong first argument type" );
-        lua_error( L );
+        lua_pushfstring(L, "addMapEvent: bad argument #1 type (uniquename as string expected, got %s!)", luaL_typename(L, 1));
+        lua_error(L);
```

Comment:
Lets keep codacy or whatever it is happy and plop in those:

    return lua_error(L);

instead of the:

    lua_error(L);
    return(1);

🤓